### DISCUSSION TITLE
Does the geocode parameter work for search_30_day and search_full_archive?

### DISCUSSION THREAD

**Discussion by igygi** [tweepy/tweepy#1460] [Questions] [answered]
Hi, 

I've been able to use the geocode parameter in the standard api.search function. But when I use it in search_30_day or search_full_archive, I cannot make it work. I get the error below:

```{'message': "There were errors processing your request: Reference to invalid field 'geocode' (at position 42), Reference to invalid operator 'geocode'. Operator is not available in current product or product packaging....}```

Here's a sample search query I pass to search ```search_30_day``` or ```search_full_archive```: ``` query = "education lang:en geocode:12.051333,123.323417,600km" ```

How do I limit the the tweet search to a particular location when doing a 30day or full-archive search?

**Comment by andypiper** [comment, marked as answer]
The Twitter premium search operators are documented [here](https://developer.twitter.com/en/docs/twitter-api/v1/tweets/search/guides/premium-operators). You will need to use `place`, `bounding_box` or `point_radius` in place of `geocode` in premium queries.

**Reply by Harmon758** [reply]
Also `place_country` if you want to search by country.
